client/src/partials/reports/modals/balance.modal.html

```diff
@@ -0,0 +1,85 @@
+<div class="modal-header">
+  <ol class="headercrumb">
+    <li class="static">{{ ReportConfigCtrl.report.title_key | translate }}</li>
+    <li class="title">{{ "FORM.LABELS.CREATE" | translate }}</li>
+  </ol>
+</div>
+
+<form name="ConfigForm" bh-submit="ReportConfigCtrl.generate(ConfigForm)" bh-form-defaults novalidate>
+<div class="modal-body">
+  <div class="form-group"
+    ng-class="{ 'has-error' : ConfigForm.$submitted && ConfigForm.label.$invalid }">
+    <label class="control-label">{{ "FORM.LABELS.LABEL" | translate }}</label>
+    <input class="form-control" id="label" name="label" ng-model="ReportConfigCtrl.label" autocomplete="off" required />
```

Comment:
Do you need autocomplete?  I think `bh-form-defaults` should take care of that.  Has something changed in Chrome that needs this?